src/main/java/io/vertx/pgclient/codec/utils/Utils.java

```diff
@@ -18,4 +18,15 @@ public static void writeCString(ByteBuf buffer, String input, Charset charset) {
     buffer.writeByte(0);
   }
 
+  public static int readSpaceCount(ByteBuf buffer) {
+    final byte SPACE = 32;
+    int spaceCount = 0;
+    for (int i = 0; i < buffer.readableBytes(); i++) {
```

Comment:
The `readSpaceCount` is counting the no of spaces in order to parse the result that depends on the SQL command for example 

INSERT  -> INSERT oid rows
UPDATE -> UPDATE rows
DELETE -> DELETE rows
SELECT -> SELECT rows

rows should be parsed as a number to get `result.getUpdatedRows()` in the Result object

I have followed the [ PostgreSQL docs in the CommandComplete section](https://www.postgresql.org/docs/9.5/static/protocol-message-formats.html)

Regarding `ByteBufProcessor` I think it does not do that for spaces.